packages/data/addon/adapters/facts/elide.ts

```diff
@@ -23,6 +23,14 @@ import { v1 } from 'ember-uuid';
 
 export const ELIDE_API_DATE_FORMAT = 'YYYY-MM-DD'; //TODO: Update to include time when elide supports using full iso date strings
 
+const escape = (value: string) => {
+  const fn = (v: string) => `${('' + v).replace(/'/g, "\\'").replace(/,/g, '\\,')}`;
+  if (Array.isArray(value)) {
```

Comment:
Do we have this case?

Reply:
For this situation I do not think so, I'll remove it